WooCommerce/Classes/Authentication/Store Creation/Free Trial/FreeTrialSummaryView.swift

```diff
@@ -0,0 +1,182 @@
+                            onClose?()
+                        } label: {
+                            Image(uiImage: .closeButton)
+                                .foregroundColor(Color(.textSubtle))
+                        }
+
+                        Spacer()
+
+                        Image(uiImage: .freeTrialIllustration)
+                    }
+                    .padding([.trailing, .bottom], Layout.illustrationInset)
+
+                    // Title
+                    Text(Localization.launchInDays)
+                        .bold()
+                        .titleStyle()
+                        .padding(.bottom, Layout.titleSpacing)
+                        .padding(.trailing, Layout.estimatedIllustrationWidth)
+
+                    // Description
+                    Text(Localization.weOfferEverything)
+                        .secondaryBodyStyle()
+                        .padding(.trailing, Layout.infoTrailingMargin)
+                        .padding(.bottom, Layout.sectionsSpacing)
+
+                    // Features
+                    Text(Localization.tryItForDays)
+                        .bold()
+                        .secondaryTitleStyle()
+                        .padding(.bottom, Layout.titleSpacing)
+
+                    ForEach(Self.features, id: \.title) { feature in
+                        HStack {
+                            Image(uiImage: feature.icon)
+                                .foregroundColor(Color(uiColor: .accent))
+
+                            Text(feature.title)
+                                .foregroundColor(Color(.text))
+                                .calloutStyle()
+                        }
+                        .padding(.bottom, Layout.featureSpacing)
+                    }
+
+                    // WPCom logo
+                    HStack {
+                        Text(Localization.poweredBy)
+                            .foregroundColor(Color(uiColor: .textSubtle))
+                            .captionStyle()
+
+                        Image(uiImage: .wpcomLogoImage(tintColor: .textSubtle))
+
+                    }
+                }
+                .padding()
+            }
+            .background(Color(.listBackground))
+
+            // Continue Footer
+            VStack() {
+                Divider()
+
+                Button {
+                    onContinue?()
+                } label: {
+                    Text(Localization.tryItForFree)
+                }
+                .buttonStyle(PrimaryButtonStyle())
+                .padding()
+
+                Text(Localization.noCardRequired)
+                    .subheadlineStyle()
+            }
+            .background(Color(.listForeground(modal: false)))
+        }
+    }
+}
+
+private extension FreeTrialSummaryView {
+    enum Localization {
+        enum Feature {
+            static let premiumThemes = NSLocalizedString("Premium Themes", comment: "Premium Themes title feature on the Free Trial Summary Screen")
```

Comment:
nit: I think only the first letter should be capitalized, to be consistent with the capitalization for other features.

```suggestion
            static let premiumThemes = NSLocalizedString("Premium themes", comment: "Premium Themes title feature on the Free Trial Summary Screen")
```